### ISSUE TITLE
German translation title

### ISSUE THREAD

**Issue by Lesserkuma** [Goppier/Gen3DistributionRoms#7]
In the German translation, the “ÄON-TICKET Wird Gesendet” text seems off. The words “Wird Gesendet” must not be capitalized, even in a title scenario. While you can capitalize titles in English, doing the same in German is not correct grammar. Having it say “ÄON-TICKET wird übertragen” would also probably sound better and look more official.

**Comment by Goppier** [comment]
For now I want to keep it capitalized the way it is. I understand it is not correct grammar, but I want the look and feel between the different languages to be similar. I will look into changing "Gesendet" to "übertragen". Thank you for your comment!

**Comment by Glurak18** [comment]
Gesendet should be gesendet and is the correct translation.  Übertragen would mean transfered.  
But it shoul say "Wird gesendet"   so lowercase not uppercase. 
I do some fan translation on other games. And i think its good how it is except it should be lowercase. Its a verb. And verbs in german are always lowercase except they start a sentence.
But its true games like Pokemon Channel for Jirachi used übertragen/überträgt.  I need to look into other stuff. https://www.youtube.com/watch?v=zE5Tb5MebLw you can see it here :)

**Comment by Lesserkuma** [comment]
Yeah, all the official stuff used the term „wird übertragen“ afaik. And writing any of these words in uppercase is still a huge no-no, unless at the beginning of a sentence, like Glurak18 said.
